src/status_im/subs/chat/chats.cljs

```diff
@@ -229,9 +229,9 @@
  :<- [:profile/multiaccount]
  :<- [:mediaserver/port]
  (fn [[contacts multiaccount port] [_ id]]
-   (let [contact (or (get contacts id)
-                     (when (= id (:public-key multiaccount))
-                       multiaccount))]
+   (let [contact (or (when (= id (:public-key multiaccount))
+                       multiaccount)
+                     (get contacts id))]
```

Comment:
Any particular reason changes the order of these two?

Reply:
contacts may contains the public key from multiaccount, but contact may only contains field: identicon , not include field: images, we should prefer show image than identicon

Reply:
In case 1, all users' avatar are broken, `multiaccount` only contains current user info. I don't think this is the root cause of case 1. Also most of the `id`s will be in `contacts` since most users in group is not the current user. I don't think it's a good idea to change the order here.

Reply:
this change is not relate to case 1, it's relate to case 2

Reply:
> this change is not relate to case 1, it's relate to case 2

I don't think this affects case 2. I can get the messages screen rendered successfully with `(re-frame/reg-sub :chats/photo-path` commented.

Reply:
sorry, i made a wrong expression, i think it's an improvement, if u edited your headicon then back to chat screen, u will c the latest headicon, otherwise u will c the old identicon @yqrashawn